h2o-docs/src/product/data-science/glm.rst

```diff
@@ -280,6 +280,106 @@ When GLM performs regression (with factor columns), one category can be left out
 The reason for the different behavior with regularization is that collinearity is not a problem with regularization. 
 And it’s better to leave regularization to find out which level to ignore (or how to distribute the coefficients between the levels).
 
+Regression Influence Diagnostics
+''''''''''''''''''''''''''''''''
+
+Regression influence diagnostics reveal the influence of each data row on the GLM parameter determination for IRLSM. This shows the parameter value change for each predictor when  a data row is included and excluded in the dataset used to train the GLM model. 
+
+To find the regression diagnostics for the Gaussian family, the output is:
+
+.. math::
+   
+   y_i = x_i^T \beta + \epsilon_i
+
+For the whole dataset, there is:
+
+.. math::
+   
+   Y = X \beta
+
+where:
+
+- :math:`Y` is a column vector with :math:`N` elements;
```

Comment:
Hannah:  Please add behind the N elements for Y as:
 and each element is :math: y_i